packages/contracts/test/core/dao/dao.ts

```diff
@@ -368,6 +374,33 @@ describe('DAO', function () {
       expect(event.args.execResults[0]).to.equal(data.successActionResult);
     });
 
+    it('reverts if failure is allowed but not enough gas is provided', async () => {
+      const GasConsumer = new GasConsumer__factory(signers[0]);
+      let gasConsumer = await GasConsumer.deploy();
+
+      const gasConsumingAction = {
+        to: gasConsumer.address,
+        data: GasConsumer.interface.encodeFunctionData('consumeGas', [20]),
+        value: 0,
+      };
+      const expectedGas = 495453; // Providing less gas causes the `to.call` of the `gasConsumingAction` to fail, but is still enough for the overall `dao.execute` call to finish successfully.
```

Comment:
where comes this number from?
I suggest you use `dao.estimateGas.execute(...)` to get a valid estimation and reduce from there to get below the 1/64 ratio

Reply:
Changed here: bf63b3d0d29ad4615440ad61e8ff18099fa70696